container-test/pom.xml

```diff
@@ -100,6 +100,12 @@
       <artifactId>commons-compress</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <!-- not used by Vespa, but was historically on test classpath -->
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <scope>compile</scope>
```

Comment:
Please don't use explicit 'compile' scope. It could imply that we manage dependency scopes, which we shouldn't.

Reply:
```suggestion
```